lib/collection/src/wal.rs

```diff
@@ -162,8 +162,7 @@ impl<'s, R: DeserializeOwned + Serialize + Debug> SerdeWal<R> {
     /// # Arguments
     ///
     /// * `until_index` - the newest no longer required record sequence number
-    ///
-    pub fn ack(&mut self, until_index: u64) -> Result<()> {
+    pub(super) fn ack(&mut self, until_index: u64) -> Result<()> {
```

Comment:
Why not just `pub`? 🪡

Reply:
We can, but I'd like to limit access to this as much as possible. We should not blindly call `ack` if this `wal_keep_from` parameter is set, because we need to consider that as well.